CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## v1.14.0 (15/11/2021)
+
+**Common**
+* **Work In Progress** Added LMDB library as a dependency. Currently not used, turned off.
+* Added handling of the `SSLException` error on Android as `IO_ERROR`.
+* Added handling of the `NSURLErrorCannotConnectToHost` error on iOS as `IO_ERROR`.
+* Added a new class `olp::cache::KeyGenerator`. It is used by the `olp-cpp-sdk-dataservice-read` module to generate cache keys for storing the data.
```

Comment:
This class will empower external users to access, insert, or remove cache entries without going through any Data SDK client.

Reply:
```suggestion
* Added the `olp::cache::KeyGenerator` class. It is used by the `olp-cpp-sdk-dataservice-read` module to generate cache keys for storing data.
```